README.md

```diff
@@ -429,8 +434,9 @@ Request parameters:
 - `currency` - the currency for this payment in 3 character ISO format, such as `GBP`
 - `order_id` - a unique order ID for this payment, sent from the client and returned by the server
 - `return_url` - where the payment form will redirect to after the payment has completed, the return url will have additional query parameters appended to indicate the status of the payment request.
```

Comment:
You should show the parameters using `camelCase`, for example `returnUrl` instead of `return_url`